traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -740,7 +758,6 @@ func scanUserRow(row *sql.Row) (tc.User, bool, error) {
 		&u.FullName,
 		&u.GID,
 		&u.ID,
-		&u.LastUpdated,
 		&u.NewUser,
 		&u.PhoneNumber,
 		&u.PostalCode,
```

Comment:
I don't think you meant to change this file

Reply:
See [comment](https://github.com/apache/trafficcontrol/pull/4633#issuecomment-617996513), fixed in 003af41431.